routes/rooms.go

```diff
@@ -56,6 +74,77 @@ func GetEmergencyRoomById(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, resp)
 }
 
+// GetEmergencyRooms godoc
+// @Summary    get all emergency rooms
+// @Tags       emergency-rooms
+// @Produce    json
+// @Param      page                            query       uint                    false   "0-indexed page number, 0 is assumed when omitted"
+// @Success    200                             {object}    GetMultipleERsResponse
+// @Failure    501                             {object}    HTTPErrorResponse
+// @Router     /er                             [get]
+func GetEmergencyRooms(ctx *gin.Context) {
+	log, logCtx := logging.GetRequestLogger(ctx)
+	db := models.GetDB(logCtx)
+
+	erPageRaw := ctx.Param("page")
+	log.Debug().Str("getting all emergency rooms on page", erPageRaw).Send()
+
+	page, err := strconv.ParseUint(erPageRaw, 10, 32)
+
+	if err != nil {
+		SendError(ctx, http.StatusBadRequest, errors.New("invalid page number"))
+		return
+	}
+
+	var er []models.EmergencyRoom
+	tx := db.Find(&er)
+
+	const pageSize = 10
```

Comment:
nit pick: I feel like constants like this should move to the top of the function, but I know that's highly debatable.